RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# CITGM Release Process
+
+This document describes the technical aspects of the citgm release process. The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.
+
+## Who can make a release?
+
+Any member of the @nodejs/citgm team can make a release.
+
+## How to create a release
+
+1. Ensure your branch is up to date with upstream
+
+```bash
+$ git remote update -p
+$ git rebase upstream/master
+$ git diff upstream/master # this should be a no-op
+```
+
+2. Install semver sync
+
+```bash
+$ npm install -g semver-sync
+```
+
+3. Bump the version and create tag with semver-sync
+
+```bash
+$ semver-sync -t patch # for patch
+$ semver-sync -t minor # for minor
+$ semver-sync -t major # for major
+```
+
+4. Push to github
+
+```bash
+$ git push origin master
```

Comment:
Shouldn't it be `upstream` ?
The new tag can be pushed in the same command with `--follow-tags` option